Author: Michael Vogt <michael.vogt@ubuntu.com>
Description: Various default settings
 Changes in various defaults.  From what I can tell (these were originally
 from ubuntu, but I (the debian maintainer) will do my best to summarize
 them below).
 .
 * Desktop Horizontal virtual size of 2 instead of 4.
 * Sync to vblank: false by default.
 * Make an exception to focus prevention matching for PolicyKit windows.
 * Remove top-right hotcorner from scale plugin by default.
 * Increase drop-shadow radius from 8 to 9.
 * Set default decorator to /usr/bin/compiz-decorator.
 * Change default colors for faces in cube plugin.
 * Use "Smart" placement mode (2) by default for placement plugin.
 * Use "Rectangle" resize mode (2) by default for resize plugin.
 * Add exception to not fade notify-osd windows in fade plugin.
 .
 The following changes from the original Ubuntu patch were modified:
 * Use slightly more Debianish/Gnomeish colors for the cube face top.
 The following changes from the original Ubuntu patch were not included:
 * Add Ubuntu logo for top face of cube.
 The following changes were introduced to the Debian version of this patch:
 * Add Debian logo for top face of cube.
Origin: vendor, ubuntu (1:0.6.0+git20071006-0ubuntu1) or perhaps earlier.
--- compiz.orig/metadata/core.xml.in.in
+++ compiz/metadata/core.xml.in.in
@@ -211,7 +211,7 @@
 		<option name="hsize" type="int">
 		    <_short>Horizontal Virtual Size</_short>
 		    <_long>Screen size multiplier for horizontal virtual size</_long>
-		    <default>4</default>
+		    <default>2</default>
 		    <min>1</min>
 		    <max>32</max>
 		</option>
@@ -284,7 +284,7 @@
 		<option name="sync_to_vblank" type="bool">
 		    <_short>Sync To VBlank</_short>
 		    <_long>Only perform screen updates during vertical blanking period</_long>
-		    <default>true</default>
+		    <default>false</default>
 		</option>
 	    </group>
 	    <group>
@@ -319,7 +319,7 @@
 		<option name="focus_prevention_match" type="match">
 		    <_short>Focus Prevention Windows</_short>
 		    <_long>Focus prevention windows</_long>
-		    <default>any</default>
+		    <default>!(class=Polkit-gnome-authentication-agent-1)</default>
 		</option>
 	    </group>
 	    <option name="unredirect_fullscreen_windows" type="bool">
--- compiz.orig/metadata/scale.xml.in
+++ compiz/metadata/scale.xml.in
@@ -27,9 +27,6 @@
 		<option name="initiate_edge" type="edge">
 		    <_short>Initiate Window Picker</_short>
 		    <_long>Layout and start transforming windows</_long>
-		    <default>
-			<edge name="TopRight"/>
-		    </default>
 		    <allowed edgednd="true"/>
 		</option>
 		<option name="initiate_key" type="key">
--- compiz.orig/metadata/decoration.xml.in
+++ compiz/metadata/decoration.xml.in
@@ -16,7 +16,7 @@
 	    <option name="shadow_radius" type="float">
 		<_short>Shadow Radius</_short>
 		<_long>Drop shadow radius</_long>
-		<default>8.0</default>
+		<default>9.0</default>
 		<min>0.1</min>
 		<max>18.0</max>
 		<precision>0.1</precision>
@@ -50,6 +50,7 @@
 	    <option name="command" type="string">
 		<_short>Command</_short>
 		<_long>Decorator command line that is executed if no decorator is already running</_long>
+		<default>/usr/bin/compiz-decorator</default>
 	    </option>
 	    <option name="mipmap" type="bool">
 		<_short>Mipmap</_short>
--- compiz.orig/metadata/cube.xml.in
+++ compiz/metadata/cube.xml.in
@@ -92,9 +92,9 @@
 		    <_short>Cube Color</_short>
 		    <_long>Color of top and bottom sides of the cube</_long>
 		    <default>
-			<red>0xfefe</red>
-			<green>0xffff</green>
-			<blue>0xc7c7</blue>
+                       <red>0x4545</red>
+                       <green>0x9a9a</green>
+                       <blue>0xf4f4</blue>     
 		    </default>
 		</option>
 		<subgroup>
@@ -109,9 +109,7 @@
 			<_long>List of PNG and SVG files that should be rendered on top face of cube</_long>
 			<type>string</type>
 			<hints>file;image;</hints>
-			<default>
-			    <value>freedesktop</value>
-			</default>
+			<default><value>/usr/share/images/desktop-base/debian-blueish-wallpaper-640x480.png</value></default>
 		    </option>
 		    <option name="adjust_image" type="bool">
 			<_short>Adjust Image</_short>
--- compiz.orig/metadata/place.xml.in
+++ compiz/metadata/place.xml.in
@@ -12,7 +12,7 @@
 	    <option name="mode" type="int">
 		<_short>Placement Mode</_short>
 		<_long>Algorithm to use for window placement</_long>
-		<default>0</default>
+		<default>2</default>
 		<min>0</min>
 		<max>5</max>
 		<desc>
--- compiz.orig/metadata/resize.xml.in
+++ compiz/metadata/resize.xml.in
@@ -40,7 +40,7 @@
 	    <option name="mode" type="int">
 		<_short>Default Resize Mode</_short>
 		<_long>Default mode used for window resizing</_long>
-		<default>0</default>
+		<default>2</default>
 		<min>0</min>
 		<max>3</max>
 		<desc>
--- compiz.orig/metadata/fade.xml.in
+++ compiz/metadata/fade.xml.in
@@ -51,7 +51,7 @@
 	    <option name="window_match" type="match">
 		<_short>Fade windows</_short>
 		<_long>Windows that should be fading</_long>
-		<default>any</default>
+                <default>any &amp; !(title=notify-osd)</default>
 	    </option>
 	    <option name="visual_bell" type="bool">
 		<_short>Visual Bell</_short>
